src/core_plugins/kibana/public/dashboard/dashboard.js

```diff
@@ -23,16 +23,23 @@ import { FilterBarClickHandlerProvider } from 'ui/filter_bar/filter_bar_click_ha
 import { DashboardState } from './dashboard_state';
 import { notify } from 'ui/notify';
 import { documentationLinks } from 'ui/documentation_links/documentation_links';
+import { showCloneModal } from './top_nav/show_clone_modal';
+import { DashboardCloneModal } from './top_nav/clone_modal';
 
 const app = uiModules.get('app/dashboard', [
   'elasticsearch',
   'ngRoute',
   'kibana/courier',
   'kibana/config',
   'kibana/notify',
-  'kibana/typeahead'
+  'kibana/typeahead',
+  'react',
 ]);
 
+app.directive('dashboardCloneModal', function (reactDirective) {
```

Comment:
We should be able to remove this as the `showCloneModal` is using the React component directly as opposed to via ng-react.